lib/output.php

```diff
@@ -135,6 +135,7 @@ public static function __callStatic(string $method, array $parameters)
+    public static $filter = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAB4AAAAeCAYAAAA7MK6iAAAACXBIWXMAAAsTAAALEwEAmpwYAAAArElEQVR4nO2SwQlCMRAFcxFbEOxAG/BiLV72vYVAerCM/9vxgx3YgWAL4kUR9CKIJpsg6g7kFhjyMiE4fwXJgeS50Rk+Jd68evXoeqmWEMA2pTR+a/IY4wTAvoL0oKrTrP8WkQWAo0F8EpFllvQOyZVBzGABQF8g7UzSktiyYqoVW1FMFWIrj8kYmy0m55FnU4fW0MX0qX85LgA7AGtVnTUX4yYTkXlzmfPVXACDguuOuS6tYAAAAABJRU5ErkJggg==';
```

Comment:
@henochit please use svg instead of png ?

Reply:
Okay, What is the benefit ?